sdk/core/azure-core/azure/core/utils/_pipeline_transport_rest_shared.py

```diff
@@ -353,10 +353,11 @@ def _aiohttp_body_helper(response):
     enc = enc.lower()
     if enc in ("gzip", "deflate"):
         if response._decompressed_content:
-            return response._decompressed_content
+            return response._content
         import zlib
         zlib_mode = 16 + zlib.MAX_WBITS if enc == "gzip" else zlib.MAX_WBITS
         decompressor = zlib.decompressobj(wbits=zlib_mode)
-        response._decompressed_content = decompressor.decompress(response._content)
-        return response._decompressed_content
+        response._content = decompressor.decompress(response._content)
```

Comment:
Is it valid that user load_body then get content?

In this case, content is still raw.

Reply:
IOW, is it possible user gets content before calling _aiohttp_body_helper?